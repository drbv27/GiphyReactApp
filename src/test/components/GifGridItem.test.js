import { shallow } from "enzyme";
import "@testing-library/jest-dom";
import { GifGridItem } from "../../components/GifGridItem";
describe("Pruebas en <GifGridItem />", () => {
  test("shouuld diplay the component right", () => {
    const wrapper = shallow(<GifGridItem />);
    expect(wrapper).toMatchSnapshot();
  });
});
