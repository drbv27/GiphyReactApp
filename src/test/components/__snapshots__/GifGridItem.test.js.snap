// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifGridItem /> shouuld diplay the component right 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img />
  <p />
</div>
`;
